config/migrations/20250204150000-migrate-old-private-comments-to-model.sparql

```diff
@@ -0,0 +1,47 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX besluitvorming: <https://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX subm: <http://mu.semte.ch/vocabularies/ext/submissions/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+     ?newInternalReview a subm:InterneBeoordeling ;
+                                              mu:uuid ?uuid ;
+                                              dct:created ?currentDateTime ;
+                                              ext:privateComment ?comment ;
+                                              subm:isInterneBeoordelingVoor ?subcase .
+  }
+} WHERE {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?agendapunt a besluit:Agendapunt;
+      ext:privateComment ?comment .
+    FILTER (STRLEN(?comment) > 2) # ignore empty properties and really short ones which are just whitespace
+    FILTER NOT EXISTS { [] prov:wasRevisionOf ?agendapunt } # final versions only so we make 1 model per subcase
```

Comment:
This one. Without it we have duplicate subcase.
But only subcases that are not listed with this filter.
So assuming that means that the final version of the agendaitem has no privateComment.
Checked some manually.
f.e. on agenda D > no private comment
on agenda C > private comment.
So it makes sense to remove those by keeping this filter and not include them.

Reply:
Makes sense. Compiling a list of these comments on production before we deploy this migration might be a good idea to at least have a record of what we did.